content/python/concepts/time-module/terms/time-ctime/time-ctime.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'time.ctime()'
+Description: 'Returns a time expressed in seconds (since the epoch) to a string format.'
```

Comment:
```suggestion
Description: 'Returns a formatted string of the time, expressed in second, since the epoch.
```
```suggestion
Description: 'Returns a time expressed in seconds (since the epoch) to a string format.'
```